core/pta/bcm/sotp.c

```diff
@@ -19,6 +19,47 @@ enum pta_bcm_sotp_cmd {
 
 #define SOTP_TA_NAME		"pta_bcm_sotp.ta"
 
+static bool sotp_access_enable = true;
+
+/**
+ * create_ta() - Print a debug message when creating a TA.
+ */
+static TEE_Result create_ta(void)
+{
+	DMSG("create entry point for static ta \"%s\"", SOTP_TA_NAME);
+	return TEE_SUCCESS;
+}
+
+/**
+ * destroy_ta() - Print a debug message when destroying a TA.
+ */
+static void destroy_ta(void)
+{
+	DMSG("destroy entry point for static ta \"%s\"", SOTP_TA_NAME);
+}
+
+/**
+ * destroy_ta() - Print a debug message when opening a TA session.
+ */
+static TEE_Result open_session(uint32_t nParamTypes __unused,
+			       TEE_Param pParams[4] __unused,
+			       void **ppSessionContext __unused)
+{
+	DMSG("open entry point for static ta \"%s\"", SOTP_TA_NAME);
+	return TEE_SUCCESS;
+}
```

Comment:
Are these 3 functions useful? You have a trace already in the only function which may do some actual work: `invoke_command()`.